Song.js

```diff
@@ -238,6 +240,44 @@ class Song
 		this.callback(this);
 	}
 
+	makeMixcloudStream(config)
+	{
+		var resolveUrl = this.url.replace("https://www.", "https://api.");
+		https.get(resolveUrl, (res) => {
+			var data = "";
+			res.setEncoding("utf8");
+			res.on("data", function(chunk) { data += chunk; });
+			res.on("end", () => {
+				if (data == "") {
+					console.log("There is no response for that URL. Resolve URL = " + urlResolve);
+					callback(this);
```

Comment:
This should be `this.callback(this);`

Reply:
Ah I merely copy pasted the code from soundcloud. I'll fix this while I'm at it